source/renderer/app/components/staking/widgets/TooltipPool-copyId-tooltip.scss

```diff
@@ -1,19 +1,17 @@
 .root {
-  .bubble {
-    margin-left: 77px;
-    top: -8px;
-    transition: opacity 0.25s;
-    transition-delay: 0s;
-    span[data-bubble-arrow='true'] {
-      &:before,
-      &:after {
-        margin-left: 45px;
-      }
+  margin-left: 230px;
+  top: 0;
+  transition: opacity 0.25s;
+  transition-delay: 0s;
+
+  .tippy-arrow {
+    &:before,
+    &:after {
+      margin-left: 45px;
```

Comment:
Can you provide some context why this was necessary? I'm just asking because there is also a Tippy prop for specifying the offsets in JS: https://atomiks.github.io/tippyjs/v6/all-props/#offset

Reply:
Removed, its obsolete now.